src/components/ContinuationIn/ExtraproRegistration.vue

```diff
@@ -182,9 +189,10 @@
             <DatePickerShared
               id="incorporation-date"
               ref="incorporationDateRef"
-              title="Date of Incorporation, Continuation or Amalgamation in Home Jurisdiction"
+              title="Date of Incorporation"
               :nudgeRight="40"
               :nudgeTop="85"
+              hint="Date of Incorporation, Continuation or Amalgamation in Home Jurisdiction"
               :persistentHint="true"
```

Comment:
For future reference, you only need `persistentHint` if you provide a (static) hint.